lxd/daemon.go

```diff
@@ -748,7 +748,7 @@ func (d *Daemon) Init() error {
 	d.lxcpath = shared.VarPath("containers")
 
 	/* Make sure all our directories are available */
-	if err := os.MkdirAll(shared.CachePath(), 0700); err != nil {
+	if err := os.MkdirAll(shared.CachePath(), 0711); err != nil {
```

Comment:
Oh, then yes, this is totally wrong in that case :). Anyway, I pushed a patch which does it for VarPath.